docs/reference/api/resources/statusicons.rst

```diff
@@ -0,0 +1,187 @@
+purposes. To put a command in a menu associated with a :class:`~toga.MenuStatusIcon`,
+set the ``group`` associated with that command, then add the command to the
+:class:`~toga.commands.CommandSet` associated with status icons. The following example
+will create a :class:`~toga.MenuStatusIcon` that has a single top-level menu item, plus
+a sub-menu that itself has a single menu item:
+
+.. code-block:: python
+
+    # Create a MenuStatusIcon
+    status_icon = MenuStatusIcon(icon="icons/blue")
+
+    # Create some commands that are associated with the # menu status icon's group.
+    def callback(sender, **kwargs):
+        print("Command activated")
+
+    cmd1 = toga.Command(
+        callback,
+        text='Example command',
+        group=status_icon,
+    )
+
+    # Create a sub-group of the status icon. This will appear as a submenu.
+    stuff_group = Group('Stuff', parent=status_icon)
+
+    cmd2 = toga.Command(
+        callback,
+        text='Stuff sub-command',
+        group=stuff_group
+    )
+
+    # Add the status icon to the app
+    app.status_icons.add(status_icon_1, status_icon_2)
+
+    # Add the commands to the status icons command set
+    app.status_icons.commands.add(cmd1, cmd2)
+
+If you add at least one :class:`~toga.MenuStatusIcon` instance to your app, Toga will
+add some standard commands to the app's status icon command set. These items will appear
+at the end of menu of the first :class:`~toga.MenuStatusIcon`'s menu. To remove these
+items, clear the app's status icon command set before adding your own commands.
+
+If you add a command to the app's status icon command set that *doesn't* belong to a
+:class:`~toga.MenuStatusIcon` group, or that belongs to a :class:`~toga.MenuStatusIcon`
+group that hasn't been registered with the app as a status icon, a ``ValueError`` will
+be raised. An error will also be raised if you *remove* a status icon while there status
+icon commands referencing that command.
+
+Notes
+-----
+
+* Status icons on GTK are implemented using the `XApp
+  <https://github.com/linuxmint/xapp>`__ library. This requires that the user has
+  installed the system packages for ``libxapp``, plus the GObject Introspection bindings
+  for that library. The name of the system package required is distribution dependent:
+
+  - Ubuntu: ``gir1.2-xapp-1.0``
+  - Fedora: ``xapps``
+
+* The GNOME desktop environment does not provide built-in support for status icons.
+  `This is an explicit design decision on their part
+  <https://blogs.gnome.org/aday/2017/08/31/status-icons-and-gnome/>`__, and they advise
+  against using status icons as part of your app design. However, there are GNOME shell
+  extensions that can add support for status icons. Other GTK-based desktop environments
+  (such as Cinnamon) *do* support status icons.
+
+Reference
+---------
+
+.. autoclass:: toga.statusicons.BaseStatusIcon
+
+.. autoclass:: toga.StatusIcon
+
+.. autoclass:: toga.MenuStatusIcon
```

Comment:
It's a little confusing that a `MenuStatusIcon` is not a `StatusIcon`. I think it would be clearer if the base class was called `StatusIcon` and the simple class was called`SimpleStatusIcon`.

Reply:
Agreed - I had the same reservation about naming.